### ISSUE TITLE
[Announce] Liquid simulation plugin

### ISSUE THREAD

**Issue by Wexelus** [liabru/matter-js#984]
**Hello everyone! Most likely, I am writing in the wrong place, but I think this is not critical :)**

I'm developing a **fluid simulation plugin** and now i have some result that i can post
At the moment, the project is still under development, and many **more interesting features are planned**.
The source code of the project is open and **you can give me recommendations for improvement**

**[Repository](https://github.com/Wexelus/matter-liquid)
[Demos](https://wexelus.github.io/matter-liquid)**

I hope this gives you a small dose of motivation to develop your beautiful engine ;)

Sorry for my Google Translater)

**Comment by liabru** [comment]
Very cool! Is there an example where it interacts with bodies? 

I remember playing with my own fluid sims a while back and found that part pretty tough.
Keep us posted, will be awesome to see where it goes.

**Comment by Wexelus** [comment]
Yes, right now I'm working on interacting with bodies, so I will publish a version with this functionality in the next week)

**Comment by Wexelus** [comment]
Finally, I can imagine the [first stage of the functionality of interaction with bodies](https://wexelus.github.io/matter-liquid/#bodies)

I decided to focus only on checking collisions with `body.verticles`, this is the most optimal solution in my opinion

**Comment by Wexelus** [comment]
**Hello again)**
Added simple fluid mass support: [Demo](https://wexelus.github.io/matter-liquid/#multi_liquid)

**Comment by Wexelus** [comment]
**こにちわ !)**
Another new major feature has been implemented: [Chemical reactions](https://wexelus.github.io/matter-liquid/#chemics) (more precisely, pseudo-chemical)

At the moment, this only works between particles, later I will implement reactions with bodies
